cluster/manifests/polarsignals/daemonset.yaml

```diff
@@ -0,0 +1,117 @@
+{{ if eq .Cluster.ConfigItems.polarsignals_enabled "true" }}
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    component: agent
+    application: polarsignals
+  name: polarsignals-agent
+  namespace: polarsignals
+  annotations:
+    node-ready.cluster.zalando.org/exclude: "true"
+spec:
+  selector:
+    matchLabels:
+      component: agent
+      application: polarsignals
+  template:
+    metadata:
+      labels:
+        component: agent
+        application: polarsignals
+    spec:
+      containers:
+        - args:
+            - /bin/parca-agent
+            - --log-level=info
+            - --node=$(NODE_NAME)
+            - --http-address=:7071
+            - --remote-store-address=grpc.polarsignals.com:443
+            - --remote-store-bearer-token-file=/var/polarsignals-agent/token
+            - --debuginfo-strip
+            - --debuginfo-temp-dir=/tmp
+            - --debuginfo-upload-cache-duration=5m
+          env:
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          image: container-registry.zalando.net/gwproxy/parca-agent:v0.41.0
+          name: polarsignals-agent
+          ports:
+            - containerPort: 7071
+              name: http
+          readinessProbe:
+            httpGet:
+              path: /ready
+              port: http
+          resources:
+            limits:
+              cpu: "{{ .Cluster.ConfigItems.polarsignals_cpu}}"
+              memory: "{{ .Cluster.ConfigItems.polarsignals_memory}}"
+            requests:
+              cpu: "{{ .Cluster.ConfigItems.polarsignals_cpu}}"
+              memory: "{{ .Cluster.ConfigItems.polarsignals_memory}}"
+          securityContext:
+            privileged: true
+            readOnlyRootFilesystem: true
+          volumeMounts:
+            - mountPath: /tmp
+              name: tmp
+            - mountPath: /run
+              name: run
+            - mountPath: /boot
+              name: boot
+              readOnly: true
+            - mountPath: /lib/modules
+              name: modules
+            - mountPath: /sys/kernel/debug
+              name: debugfs
+            - mountPath: /sys/fs/cgroup
+              name: cgroup
+            - mountPath: /sys/fs/bpf
+              name: bpffs
+            - mountPath: /var/run/dbus/system_bus_socket
+              name: dbus-system
+            - mountPath: /var/polarsignals-agent
+              name: token
+      hostPID: true
+      serviceAccountName: polarsignals-agent
+      nodeSelector:
+        kubernetes.io/os: linux
+        dedicated: skipper-ingress
```

Comment:
why this needs to run on dedicated nodepools?

Reply:
In the pilot phase it will be limited to only the skipper-ingress use case.